data/human/human missions.txt

```diff
@@ -6890,3 +6890,1716 @@ mission "Hogshead's Dilemma II"
+
+
+mission "Timothy Radrickson 2: Interlude"
+	landing
+	to offer
+		has "Timothy Radrickson 1: Stowaway: failed"
+		has "timothy radrickson: port"
+	source
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
+	on offer
+		conversation
+			`Blinking at the light Timothy shuffles toward the exit. "Th-th-thank you. Thank you so much," Timothy mumbles.`
+			`	Timothy opens the hatch and looks back at you. "You s-s-saved my life. I won't forget. I won't."`
+			`	With that Timothy exits your ship and begins his new life. You hope everything works out for the best.`
+
+
+mission "Timothy Radrickson 2: Good Interlude"
+	landing
+	invisible
+	to offer
+		has "event: timothy radrickson: on rand"
+	on offer
+		payment 50000
+		event "timothy radrickson: governor" 3600
+		conversation
+			`After your ship touches down you notice a light blinking on your dashboard. You have a cached message waiting for you to play.`
+			choice
+				`	(Play the message.)`
+
+			`	The screen lights up with the familiar, fuller face of a healthy Timothy Radrickson.`
+			`	"Captain <last>. Let me just reiterate my thanks to you for what you did for me. Most captains would have airlocked me, or dropped me off wherever, but you really gave me a chance."`
+			`	He stops and grins at the camera for a moment. "And that chance, it really has made all the difference. It's been a year of hard work, but I've saved some money and made some investments, and things really are turning around."`
+			`	"As way of thanks," he continues, "I've included 50,000 credits with this message. That should cover the cost of transporting to me to Rand, but it doesn't come to close to what I really owe. Which is, in all honesty, my very life."`
+			`	He pauses, as if he's trying to remember something. "Anyways, thank you again, and know I'll never forget what you did for me."`
+			`	Timothy waves and then the video ends.`
+				decline
+
+
+event "timothy radrickson: governor"
+
+
+# Plan:
+# Tim invites the PC to come and visit him on Rand. He takes you to his governor palace, thanks you for all you did, gives you 125,000 for giving him a ride all those years ago.
+# He talks about how Rand just sent out the largest mineral freighter fleet ever to New Britain, and when it returns the colony will finally have enough money to start taking care of it self...
+# When.. uh oh the frieghters have lost communication or sent out a distress signal or something, and you offer to help.
+# You fly off, find a marauder/pirate fleet, destroy them, save the freighter fleet, go back to Rand and then Tim holds a swanky dinner and names you Hero of Rand.
+
+
+mission "Timothy Radrickson 3: Governor Notification"
+	name "Visit the Governor on <planet>"
+	description "The Governor of <planet> has invited you to speak with them."
```

Comment:
```suggestion
	description "The governor of <planet> has invited you to speak with them."
```
I don't think the cap's required.

Reply:
You're referring to a specific governor here, not just governors in general. I believe that is a criterion for capitalising it.